app/controllers/AssessorController.scala

```diff
@@ -42,7 +42,7 @@ trait AssessorController extends BaseController {
   }
 
   def addAvailability(userId: String): Action[JsValue] = Action.async(parse.json) { implicit request =>
-    withJsonBody[AssessorAvailability] { availability =>
+    withJsonBody[AssessorAvailabilityOld] { availability =>
```

Comment:
I don't think we should be naming anything *Old? If it's old it should be removed i think?